src/dev.ts

```diff
@@ -78,19 +71,25 @@ appHost.start({
     },
     skipUnknownEvents: false,
     reset: true,
-    blockIndexer: "amur",
     network: "polygon-mumbai",
+    stateManager: "state-manager-local",
     chainlistId: "80001",
     startBlock: "26224255",
     initialOffset: 26224255,
-    outputStream: "domain-events",
-  },
-  source: {
-    db: "kafka-amur",
-    streams: ["polygon-mumbai-blockindex"],
-  },
-  target: {
-    db: "kafka-dev",
-    namespace: id,
+
+    input: {
+      default: "polygon-mumbai-blockheader",
+    },
+    output: {
+      default: "domain-events",
+    },
+
+    db: "streamdb-01",
   },
-});
+};
+
+appHost.start(mangroveStrategiesApp);
+
+if (false) {
+  appHost.start(parseBlocksApp);
```

Comment:
You can just name it `_parseBlocksApp` if you are just trying to avoid the warning

Reply:
Not working for me: "62:7  warning  '_parseBlocksApp' is assigned a value but never used  @typescript-eslint/no-unused-vars
"